### PR TITLE
NurseConnect post Nurse registration to Control

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: src/utils.js [hunk 1]
```diff
@@ -661,6 +661,23 @@ go.utils = {
             });
     },
 
+    post_nursereg: function(contact, im) {
+        var payload = {
```
Comment (imsickofmaps) [inline src/utils.js:665]: Thoughts on whether this should be ready to accept the additional fields if present? Or saving for later?

**Comment by imsickofmaps** [comment]
Minor query but :+1:

File: src/utils.js (hunk 1 above)
Comment (gsvr) [inline src/utils.js:665]: Will deal with this with change handling - will need to implement changes in django as well.

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**